### PR TITLE
docs: Add warning message to lwc-local-dev 

### PR DESCRIPTION
**Pull request by BrettMN**
### What does this PR do?

This update to warn potential users on the limitations of the LWC Local Dev Server and request they provide feedback on the issue.

### What issues does this PR fix or reference?
This PR references the request for feedback on the LWC Local Dev Server and issues that prevent it from working.
#5165

### CODE REVIEW

File: docs/_articles/en/localdev/lwclocaldev.md [hunk 1]
```diff
@@ -5,6 +5,19 @@ lang: en
...
+> This plugin is out of date and does not support the following features:
+> - `@AuraEnabled` Apex methods
+> - CSS custom properties
+> - LightningModal
+> - lwc:if, lwc:elseif, lwc:else directives
+> - multiple directories and sfdx multiple packages
+> 
+> The best use you can get out of it is some light wireframing.  
+>
+> If this modual has been useful to you in the past or would like to be able to use it in the furture please be sure to provide feadback on the issue [LWC Dev Server Request for feedback](https://github.com/forcedotcom/lwc-dev-server-feedback/issues/143).
```
Comment (AnanyaJha) [inline docs/_articles/en/localdev/lwclocaldev.md:18]:
```suggestion
> If this feature has been useful to you in the past or you would be interested in this functionality, please be sure to provide feedback on the issue [LWC Dev Server Request for feedback](https://github.com/forcedotcom/lwc-dev-server-feedback/issues/143). 
```

**Comment by AnanyaJha** [comment]
Hi @BrettMN ! Thank you for taking the time to make this update, we'll get this reviewed and merged. @sbudhirajadoc @gbockus-sf

**Comment by sbudhirajadoc** [comment]
I made some editorial updates. Can you please review, @AnanyaJha ? Thank you!

File: docs/_articles/en/localdev/lwclocaldev.md [hunk 2]
```diff
@@ -5,6 +5,15 @@ lang: en
 
 The Local Development Server is a Salesforce CLI plug-in that configures and runs a Lightning Web Components-enabled server on your computer. You can develop Lightning web components and see live changes without publishing the components to an org.
 
+**Note**: This plugin does not support the following features:
+- `@AuraEnabled` Apex methods
+- `CSS` custom properties
+- `LightningModal`
+- `lwc:if`, `lwc:elseif`, and `lwc:els`e directives
```
Comment (AnanyaJha) [inline docs/_articles/en/localdev/lwclocaldev.md:12]:
```suggestion
- `lwc:if`, `lwc:elseif`, and `lwc:else` directives
```

**Review by sbudhirajadoc — APPROVED** [review]

**Pull request merged**